src/einsteinpy/tests/test_symbolic/test_ricci.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import sympy
+
+from einsteinpy.symbolic import MetricTensor, RicciTensor
+
+
+def spherical_metric():
+    symbolstr = "r theta phi"
+    syms = sympy.symbols(symbolstr)
+    list2d = (np.zeros(shape=(3, 3), dtype=int)).tolist()
+    list2d[0][0] = 1
+    list2d[1][1] = syms[0] ** 2
+    list2d[2][2] = (syms[0] ** 2) * (sympy.sin(syms[1]) ** 2)
+    metric = MetricTensor(list2d, syms)
+    return metric
+
+
+def test_Ricci_is_zero_for_spherical_space():
+    metric = spherical_metric()
+    dims = metric.dims
+    T = RicciTensor.from_metric(metric)
+    arr = T.tensor()
+    for i in range(dims ** 4):
+        n = i % dims
+        r = (int(i / dims)) % (dims)
+        s = (int(i / (dims ** 2))) % (dims)
+        t = n
+        assert arr[t, s, r, n] == 0
```

Comment:
you forgot to change the test as discussed in the meeting.
```
for i in range(dims ** 2):
    n = i % dims
    r = (int(i / dims)) % (dims)
    assert arr[n, r] == 0
```

Reply:
I was thinking about writing this..... but I did r , s   They would be wrong, right?
def test_Ricci_is_zero_for_spherical_space():
    metric = spherical_metric()
    dims = metric.dims
    T = RicciTensor.from_metric(metric)
    arr = T.tensor()
    for i in range(dims ** 2):
        r = (int(i / dims)) % (dims)
        s = (int(i / (dims ** 2))) % (dims)
        assert arr[s, r] == 0

Reply:
def test_Ricci_is_zero_for_spherical_space():
metric = spherical_metric()
dims = metric.dims
T = RicciTensor.from_metric(metric)
arr = T.tensor()
for i in range(dims ** 2):
r = (int(i / dims)) % (dims)
s = (int(i / (dims ** 2))) % (dims)
assert arr[s, r] == 0

I have some mistakes in r=... and s=....

Reply:
with this `s`, `s = (int(i / (dims ** 2))) % (dims)`, `s` would never assume any value other than 0.
Whatever code snippet I provided above is correct I guess, in accordance to my understanding.
:)

Anyways, try to print the indices on you local machine, you would know what is wrong!

Reply:
Just a small function to check if you still have doubt
```
>>> def testfunc(dims):
...     for i in range(dims**2):
...             p = i%dims
...             q = int(i/dims)%dims
...             print(p,q)
...
>>> testfunc(3)
0 0
1 0
2 0
0 1
1 1
2 1
0 2
1 2
2 2
```